notebooks/examples/basic/custom_check_example.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.                fig.suptitle(f'Disterbution of {self.selected_feature} [{min_val:.2}, {max_val:.2}]', fontsize=16)
</pre><p>"Distribution"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>